tf2_kdl/package.xml

```diff
@@ -12,13 +12,13 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_depend>eigen</build_depend> <!-- needed by kdl interally -->
+  <build_depend>eigen3</build_depend> <!-- needed by kdl interally -->
   <build_depend>orocos_kdl</build_depend>
   <build_depend>tf2</build_depend>
   <build_depend>tf2_ros</build_depend>
 
   <run_depend>cmake_modules</run_depend> <!-- needed for eigen -->
-  <run_depend>eigen</run_depend>
```

Comment:
Eigen is Eigen3 by default. 
https://index.ros.org/d/eigen/
If you want eigen2 you call it out explicitly
https://index.ros.org/d/eigen2/

But I think we've effectively transtioned to eigen3 fully now based on the rosindex lack of usage of the older key.

Reply:
Understood. I recall having had some issues with that in OS X but can't reproduce it right now so I'll return to `eigen` in the package.xml.